packages/terra-grid/CHANGELOG.md

```diff
@@ -0,0 +1,71 @@
+Terra Grid Changelog
+====================
+
+3.1.0 - 2017-06-20
+------------------
+### Enhancements
```

Comment:
Would it be helpful consumers to separate dependencies from enhancements by listing dependency changes under it's own section. Then a consumer could quickly see dependency changes verse enhancements in functionality?